theme/templates/accounts/profile.html

```diff
@@ -741,40 +747,40 @@ <h3 class="panel-title"><span
+                                                            <span class="title"><a
+                                                                    href="{{ res.get_absolute_url }}">{{ res.metadata.title }}</a></span>
+                                                            <h5><span class="text-muted">Created: </span>{{ res.created }}
+                                                            </h5>
+                                                            <h5><span class="text-muted">Authors: </span>
+                                                                {% for creator in res.metadata.creators.all %}
+                                                                    {% if forloop.counter0 != 0 %}<span> · </span>
+                                                                    {% endif %}
+                                                                    {% if creator.description %}
+                                                                        <a href="{{ creator.description }}">{{ creator.name }}</a>
+                                                                    {% else %}
+                                                                <span>{{ creator.name }}</span>
+                                                            {% endif %}
+                                                        {% endfor %}
+                                                    </h5>
                                                 </div>
-                                                <div class="btn-show-more toggle-abstract btn btn-default btn-xs">Show More</div>
-                                            {% endif %}
 
+                                                {% if res.metadata.description %}
+                                                    <div class="activity-description col-sm-12">
+                                                        <p>
+                                                            <strong>ABSTRACT: </strong>{{ res.metadata.description|linebreaks }}
+                                                        </p>
+                                                    </div>
+                                                    <div class="btn-show-more toggle-abstract btn btn-default btn-xs">Show More</div>
+                                                {% endif %}
+                                            </div>
                                         </div>
-                                    </div>
-                                    {% endwith %}
-                                {% endfor %}
-                                {% else %}
-                                    <span class="text-muted">This user has not shared any resources.</span>
+                                        {% endwith %}
+                                    {% endfor %}
+                                   <span class="text-muted">End of Resource Listing</span>
```

Comment:
This message won't show up if the user owns no resources at all. Furthermore, I think a message that clearly states no resources were found is more appropriate. 

The right way to address the issue would be adding the message "No results found." that shows up when the filter returns no resources.